core/services/fluxmonitor/flux_monitor.go

```diff
@@ -285,13 +290,24 @@ func (f pollingDeviationCheckerFactory) New(
 		return nil, err
 	}
 
+	var flagsContract *contracts.Flags
+	if f.store.Config.FlagsContractAddress() != "" {
+		flagsContractAddress := common.HexToAddress(f.store.Config.FlagsContractAddress())
+		flagsContract, err = contracts.NewFlagsContract(flagsContractAddress, f.store.EthClient)
+		if err != nil {
+			panic(fmt.Sprintf("unable to start flux monitor, error creating Flags contract: %v", err))
```

Comment:
I was hesitant to just log an error here, because if a node op accidentally adds the wrong address for the flags contract, they could miss hibernation / reactivation logs and that could be bad news. I think it would be better to fail loud and fast. But maybe that's not worth crashing the node,,, maybe just a log and an async error would do the job?

Reply:
changed this to just log the error